### PR TITLE
[somfymylink] add missing feature.xml and pom entry

### LINKED ISSUE #8008: Somfy myLink Binding missing in 2.5.6 feature.xml
The add-on Somfy myLink (Binding #5652) is missing a `feature.xml` file and this prevents the binding from showing up in both paperUI and the console.

It's also missing in the `bom/openhab-addons/pom.xml`, that should be fixed too, but has no effect on runtime of the binding.

### PR DESCRIPTION
**Pull request by LoungeFlyZ**
[somfymylink] add missing feature.xml and pom entry

Adds missing feature.xml file and entry in pom.xml.  

Fixes #8008

### CODE REVIEW

File: bundles/org.openhab.binding.somfymylink/src/feature/feature.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.somfymylink-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-somfymylink" description="Somfy MyLink Binding" version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.somfymylink/${project.version}</bundle>
+	</feature>
+</features>
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.somfymylink/src/feature/feature.xml:9]: Has an additional space at the end and missing a newline

Comment (Hilbrand) [inline bundles/org.openhab.binding.somfymylink/src/feature/feature.xml:9]:
```suggestion
</features>

```

**Comment by Hilbrand** [comment]
Someone complained it still didn't work. After having a new look I noticed the feature file is in the wrong directory. It should be in `src/main/feature/`

**Comment by LoungeFlyZ** [comment]
Argh. Sorry about this. I'll submit an update.

**Pull request merged**